middleware/AuthMiddleware.js

```diff
@@ -0,0 +1,27 @@
+import tokenService from '../services/tokenService';
+
+/**
+ * Authentication middleware
+ */
+class AuthMiddleware {
```

Comment:
We need to agree on a consistent way of writing our middlewares. Either to use static methods or `functions`. Over here are using  class static methods as middleware in another place we are using functions

Reply:
Done